app/server/tests/functions/communityFunctions.js

```diff
@@ -0,0 +1,66 @@
+require('dotenv').config();
+const supertest = require('supertest');
+const endpoint = process.env.API_REF;
+const request = supertest(endpoint);
+
+function getCommunities(token){
```

Comment:
I really like how simple these functions are. Easy to understand.

Reply:
thanks, we will do this with all the requests that are documented on swagger from now on